samples/dect/dect_phy/dect_shell/src/dect/mac/dect_phy_mac_shell.c

```diff
@@ -458,15 +466,22 @@ static int dect_phy_mac_rach_tx_cmd(const struct shell *shell, size_t argc, char
 	if (argc < 2) {
 		goto show_usage;
 	}
+	if (argv[1] != NULL && !strcmp(argv[1], "stop")) {
+		dect_phy_mac_ctrl_rach_tx_stop();
+		desh_print("rach_tx stopped.");
+		return 0;
```

Comment:
I'd think of having `-s|--stop` option for stopping to make it more compatible with what we have. I guess this is the first time we (mosh or desh) have an argument (`argv[1]` in this case) that is first compared with string comparison and then a fall back to `getopt` done.
If you ever need to have arguments with stop, it will be inconvenient, although possible.

Reply:
Not sure do I understand. Current dect ping/rf_tool/rss_scan/perf are having like this is now in PR. for this command I'd rather keep it like 
 this in PR.

Reply:
True, seems this is more or less common in DeSh so not worth doing anything at least within the context of this PR.